message.module

```diff
@@ -15,10 +16,10 @@ use Drupal\message\Entity\MessageType;
  *
  * Handles messages deletion when referenced entities are being deleted.
  *
- * @param \Drupal\Core\Entity\EntityInterface $entity
```

Comment:
I'm pretty sure there is no need since we've added it to `use` - https://github.com/Gizra/message/pull/97/files#diff-e1850e575cf08eedf0ba88d0179db1b2R8